web/src/layouts/TopAppBar/TopAppBar.tsx

```diff
@@ -7,7 +7,11 @@ import { useUserContext } from "@features/UserProvider/useUserContext";
 import { NavLink } from "react-router-dom";
 import { Paths } from "@routing/paths";
 
-export const TopAppBar: React.VFC = () => {
+interface Props {
+  showLogo?: boolean;
+}
+
+export const TopAppBar: VFC<Props> = ({ showLogo = true }) => {
```

Comment:
Could you point me to the place, where some logo is displayed? Thanks

Reply:
I think that it is going to be on main index page. I don't have it on figma views with which I am working and didn't want to break anything anywhere that is why I set it to be true by default and pass false by props in my components.